RPIPs/RPIP-47.md

```diff
@@ -1,22 +1,25 @@
 ---
-rpip:
-title: Remove Operator-gated Upgrades
+rpip: 47
+title: Enable Forced Delegate Upgrades
 description: Allow for the protocol to make protocol-wide decisions rather than relying on per-operator upgrades
 author: Valdorff (@Valdorff)
 discussions-to: TBD
 status: Draft
 type: Protocol
 category: Core
 created: 2024-03-08
+requires: 43
 ---
 
 ## Abstract
 Currently, Node Operators can choose to upgrade (or not) for their minipool delegate (the contract at the Ethereum withdrawal address that governs how funds are disbursed, among other things). This (a) means the protocol can only effect change in ways that are in the interest of individual NOs, and (b) makes it more challenging to design the protocol as it must remain backwards compatible with every past minipool delegate.
 
-This proposal suggests removing that choice in the future. Instead, a time lock will provide time for users to initiate withdrawals in a case where they are unwilling to operate under a new incoming set of rules.
+This proposal suggests limiting that upgrade choice in the future. Users will be able to opt in to upgrade or use an "old" delegate for a defined period after a newer version. However, once the defined period has passed, they will no longer use the old delegate.
 
 ## Specification
-- Contracts moving forward SHALL NOT require Node Operators to individually opt in to them
+- Megapool delegate contracts SHALL have an expiration block
```

Comment:
Could clarify this is about an on-chain block number, not e.g. a 'block on expiration'